reader-shp/src/main/java/com/graphhopper/reader/shp/Utils.java

```diff
@@ -0,0 +1,17 @@
+package com.graphhopper.reader.shp;
+
+import com.graphhopper.util.PointList;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.GeometryFactory;
+
+public class Utils {
+	public static String toWKT(PointList list) {
```

Comment:
Is it necessary to create a new factory for every call? Or is performance overhead neglectable and problematic stuff to ugly like non-thread safety?